semgrep/tests/e2e/test_ci.py

```diff
@@ -135,29 +133,20 @@ def ci_mocks(base_commit, autofix):
         """
     ).lstrip()
 
-    with mock.patch.object(
-        GitlabMeta, "_fetch_branch_get_merge_base", return_value=base_commit
-    ):
-        # Mock rules
-        with mock.patch.object(
-            ConfigPath, "_make_config_request", mock.Mock(return_value=file_content)
-        ):
-            # Mock deployment_id
-            with mock.patch.object(
-                ScanHandler,
-                "_get_deployment_details",
-                mock.Mock(return_value=(DEPLOYMENT_ID, "org_name")),
-            ):
-                with mock.patch.object(auth, "is_valid_token", return_value=True):
-                    with mock.patch.object(
-                        ScanHandler, "autofix", mock.PropertyMock(return_value=autofix)
-                    ):
-                        with mock.patch.object(AppSession, "post"):
-                            yield
+    mocker.patch.object(ConfigPath, "_make_config_request", return_value=file_content)
+    mocker.patch.object(
+        ScanHandler, "_get_deployment_details", return_value=(DEPLOYMENT_ID, "org_name")
+    )
+    mocker.patch("semgrep.app.auth.is_valid_token", return_value=True)
```

Comment:
When we patch a member of a module (like here, it's a function defined on the top level). We can use this form instead of `mocker.patch.object` so that we don't need to import. It's kinda annoying that this way IDEs will not automatically update import paths, so maybe this is worse though.